JeMPI_Apps/JeMPI_UI/docker-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+# Set default variables 
+
+export REACT_APP_NODE_ENV=${REACT_APP_NODE_ENV:-"development"}
+export REACT_APP_MOCK_BACKEND=${REACT_APP_MOCK_BACKEND:-"false"}
+export REACT_APP_JEMPI_BASE_URL=${REACT_APP_JEMPI_BASE_URL:-"http://localhost:50000"}
```

Comment:
Since we are already using a shell script, should we not do `http://$(hostname -I):5000`

Reply:
Hi @walisc, this file was refactored.